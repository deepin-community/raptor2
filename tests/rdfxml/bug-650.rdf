<?xml version="1.0"?>
<rdf:RDF xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
         xmlns:ns="http://example.org/ns1">
  <rdf:Description rdf:about="http://www.example.org/foo">
    <rdf:value parseType="Literal">
      <ns: rdf:=""	xmlns:dc="p" dc:="" />
    </rdf:value>
  </rdf:Description>
</rdf:RDF>
