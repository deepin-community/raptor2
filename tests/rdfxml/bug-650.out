<http://www.example.org/foo> <http://www.w3.org/1999/02/22-rdf-syntax-ns#value> "\n      <ns: xmlns:dc=\"p\" xmlns:ns=\"http://example.org/ns1\" xmlns:rdf=\"http://www.w3.org/1999/02/22-rdf-syntax-ns#\" dc:=\"\" rdf:=\"\"></ns:>\n    "^^<http://www.w3.org/1999/02/22-rdf-syntax-ns#XMLLiteral> .
